05-visualisation-ggplot2.Rmd

```diff
@@ -152,7 +152,7 @@ ggplot(surveys_complete)
 - define aestetics (`aes`), that maps variables in the data to axes on the plot
      or to plotting size, shape color, etc.,
 
-```{r}
+```{r, eval=FALSE}
```

Comment:
Right, it's in the code to demonstrate that it won't work. So, it's not supposed to work.